### PR TITLE
[Azure Pipelines][Minor] Adding stage for win32

### PR DESCRIPTION
**Pull request by SukruthGrandhiNI**
- [ ]~~This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).~~

- [ ]~~I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.~~

- [ ]~~I've added tests applicable for this pull request~~

### What does this Pull Request accomplish?
In the earlier [PR](https://github.com/ni/nimi-python/pull/1653), I added pipeline stage for running system tests on the platform win64(64-bit Python interpreter on a 64-bit Windows 10 system).
In this PR, I have added new stage for win32(32-bit Python interpreter on a 64-bit Windows 10 system).

Features
 1) Both the platform stages are run in parallel.
 2) 4 agents are created(NIMI-BOT-WIN32-1, NIMI-BOT-WIN32-2, NIMI-BOT-WIN32-3, NIMI-BOT-WIN32-4) in the [nimi-bot](https://ni.visualstudio.com/_settings/agentpools?poolId=474&view=agents) agents pool.
3) All the 4 agents are hosted on the single VM which is hosted in MTO NIB Server (inside NI).

### What testing has been done?
Ran the pipeline for 3 times
https://ni.visualstudio.com/Users/_build/results?buildId=1367689&view=results
https://ni.visualstudio.com/Users/_build/results?buildId=1367696&view=results
https://ni.visualstudio.com/Users/_build/results?buildId=1367759&view=results

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -34,14 +34,29 @@ pr:
 stages:
            
   - stage: system_tests_win64
+    dependsOn: []
     pool:
       name: nimi-bot
       demands:
         - nimi-bot -equals True
-        - agent.os -equals Windows_NT
+        - platform -equals win64 # Run system tests using python 64 bit installed on windows 64 bit
```
Comment (marcoskirsch) [inline azure-pipelines.yml:42]: Mind spelling - this is a public facing repository.

```suggestion
        - platform -equals win64 # Run system tests using 64-bit Python interpreter on a 64-bit Windows 10 system.
```

File: azure-pipelines.yml [hunk 2]
```diff
@@ -34,14 +34,29 @@ pr:
...
+      name: nimi-bot
+      demands:
+        - nimi-bot -equals True
+        - platform -equals win32 # Run system tests using python 32 bit installed on windows 64 bit
+    jobs:
+    - ${{each module_name in parameters.module_names}}:
+      - template: run_module_system_tests.yml
+        parameters:
+          module_name: ${{module_name}}
+          platform: win64
```
Comment (marcoskirsch) [inline azure-pipelines.yml:62]: was this supposed to say `win32`

File: azure-pipelines.yml [hunk 3]
```diff
@@ -34,14 +34,29 @@ pr:
...
           module_name: ${{module_name}}
           platform: win64
+
+  - stage: system_tests_win32
+    dependsOn: []
+    pool:
+      name: nimi-bot
+      demands:
+        - nimi-bot -equals True
+        - platform -equals win32 # Run system tests using python 32 bit installed on windows 64 bit
```
Comment (marcoskirsch) [inline azure-pipelines.yml:56]: see suggestion for line 42

**Review by marcoskirsch — CHANGES_REQUESTED** [review]

**Comment by marcoskirsch** [comment]
test this please

File: azure-pipelines.yml (hunk 2 above)
Comment (marcoskirsch) [reply azure-pipelines.yml:62]: also note you probably should have a newline afterwards.

Comment (SukruthGrandhiNI) [reply azure-pipelines.yml]: I have removed platform from the run_module_system_tests.yml file

**Comment by SukruthGrandhiNI** [comment]
test this please

**Review by marcoskirsch — APPROVED** [review]

**Pull request merged**